src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let result = initialValue;
```

Comment:
```suggestion
    let accumulator = initialValue;
```
This variable 'accumulates' all results, so it would be better to call it in such way